tests/test_filter.py

```diff
@@ -11,24 +11,38 @@ def test_simple_language():
                                                  'category': 'profile'}
 
 
+def test_not_in_languages():
```

Comment:
Please add test for `verified and operator == InclusionOperator.NOT_IN` case

Reply:
It was in `test_verified_language_not_in_is_incorrect`. Anyway, I reorganised and complemented tests